### ISSUE TITLE
Compiling tutorial needs an update

### ISSUE THREAD

**Issue by ReinerBforartists** [Bforartists/Bforartists#221]
Our compiling tutorial for Windows is a bit aged now. A few things have changed. A few things are missing.
So we need to update our tutorial.

http://www.bforartists.de/data/tuts/Building_Bforartists_with_Cmake_and_VS_2013_at_Windows_7.pdf

**Comment by ReinerBforartists** [comment]
The editable odt version is here: 
[Building_Bforartists_with_Cmake_and_VS_2013_at_Windows_7.zip](https://github.com/Bforartists/Bforartists/files/2134069/Building_Bforartists_with_Cmake_and_VS_2013_at_Windows_7.zip)

**Comment by Draise14** [comment]
Just curious, to modify this to say... fix the typo - would it be downloading from here, then uploading here in this thread again?

**Comment by ReinerBforartists** [comment]
Yes you can :)

Hm, i could make a repo for our tutorials too. But currently it's basically just this tutorial here. So ... ^^

**Comment by Draise14** [comment]
Dibs, I'll fix the typo and also the link and description for Visual Studio Community 2013.

**Comment by Draise14** [comment]
[Building_Bforartists_with_Cmake_and_VS_2013_at_Windows_7.zip](https://github.com/Bforartists/Bforartists/files/2134394/Building_Bforartists_with_Cmake_and_VS_2013_at_Windows_7.zip)

Fixed: 
- Typo on "Wine64" fixed. Now correct as "Win64"
- Link to **Visual Studio 2013 Community** updated
- Removed comment about **Visual Studio 2013 Express** possibly being ok to avoid confusion.

**Comment by ReinerBforartists** [comment]
Thanks :)

I'll wait until you are through with proper compiling, and everything runs fine at your end. Then i will try to implement your personal experience into the tutorial too. You can add some notes too if you want :)

**Comment by ReinerBforartists** [comment]
Draise, does it compile fine now for you? Then i could have a look at this here.

**Comment by Draise14** [comment]
I haven't uninstalled and reinstalled the CUDA version yet to troubleshoot. I'll do that today and give it a whirl. I'll tag this and update the tutorial a little, yeah?

**Comment by Draise14** [comment]
I have donethe updates. 

I added quick tips that I've learnt and also updated the links. I've also made it more methodological to follow easier - and added a touch of entertainment. I have also grouped and classified the software to understand what they are all for. I have also added a quick overview and metaphor to make it easy to understand what the compiling process is and to give the brain a good overview before getting into the details. I also fixed a lot of grammatical mistakes and inconsistencies. 

[Building_Bforartists_with_Cmake_and_VS_2013_on_Windows - ver3.zip](https://github.com/Bforartists/Bforartists/files/2234199/Building_Bforartists_with_Cmake_and_VS_2013_on_Windows.-.ver3.zip)
